CONTRIBUTING.md

```diff
@@ -1,25 +1,49 @@
+# Guidelines for Contributing to CraftOS-Standards
 
-## Guidelines for Contributing to CraftOS-Standards
-
-Welcome, fellow developer! First off, make yourself familiar with the [tutorial](./Tutorial/GettingStarted.md). It's a long read, so enjoy it with some :coffee:. If you'd like to create new standards, the tutorial should actually give you all that you'll need to know. Thanks for helping us out by standardizing something awesome!
+Welcome, fellow developer! First off, make yourself familiar with the [tutorial](./Tutorial/GettingStarted.md). It's a
+long read, so enjoy it with some :coffee:. If you'd like to create new standards, the tutorial should actually give you
+all that you'll need to know. Thanks for helping us out by standardizing something awesome!
 
 ### A Note on Licensing
-All standards published here on CraftOS-Standards are licensed under the [GNU Free Documentation License](../LICENSE.md). **This includes all code examples**. For the sake of open source software and simplicity alike, libraries for dealing with your standard should also be licensed under [similar FOSS licenses](http://choosealicense.com/). After all, licensing official utilities for an open source standard under a closed source license wouldn't make much sense.
+All standards published here on CraftOS-Standards are [licensed](/LICENSE.md) under
+the [GNU Free Documentation License](https://www.gnu.org/licenses/fdl-1.3.en.html). **This includes all code
+examples**. For the sake of open source software and simplicity alike, libraries for dealing with your standard should
+also be licensed under [similar FOSS licenses](http://choosealicense.com/). After all, licensing official utilities for
+an open source standard under a closed source license wouldn't make much sense.
 
 ## Getting Started: Becoming a CraftOS-Standards Contributor
 
-If, for some reason, you'd like to join our amazing [contributor team](https://github.com/oeed/CraftOS-Standards/graphs/contributors), you will need to know a few more things about how this repository works. Great news! We wrote them down here so that you won't have to bother us about the details.
+If, for some reason, you'd like to join our
+amazing [contributor team](https://github.com/oeed/CraftOS-Standards/graphs/contributors), you will need to know a few
```

Comment:
"Contributors" actually means everyone who has committed to the repo. Collaboratorship is the word GitHub uses for people with write access.